gulpfile.js/changelog.js

```diff
@@ -100,14 +100,16 @@ async function getLog(range) {
 }
 
 const revisionRanges = {
-	nextRelease: git.raw(['describe', '--abbrev=0', '--tags']).then(res => `${res.trim()}..HEAD`)
```

Comment:
I have no idea why but this crashed the build with a

```
fatal: No names found, cannot describe anything.
``` 

No other information given. 

It worked before, so my guess is that either GH made a slight change to how PRs are setup on workflows or that webfont or one of its dependencies messes with git in some way. This problem doesn't occur locally.

No idea what the error is or why it happened but I fixed it.